scripts/badger/guestlist.py

```diff
@@ -0,0 +1,69 @@
+from rich.console import Console
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import get_registry as registry
+from helpers.constants import MaxUint256, AddressZero
+
+C = Console()
+
+# assuming techOps
+r = registry()
+SAFE = GreatApeSafe(r.badger_wallets.techops_multisig)
+SAFE.init_badger()
+
+# Add the vaults addresses to remove caps from
+VAULTS_ARRAY = []
+
+# Sets a new user cap for a given vault
+def set_vault_user_cap(vault, cap):
+    vault = SAFE.contract(vault)
+    guestlist = vault.guestList()
+    if guestlist != AddressZero:
+        C.print(f"Guestlist: {guestlist}")
+        SAFE.badger.set_guestlist_user_cap(guestlist, cap)
```

Comment:
will advise in the script to only post the tx within the *if* code block, similarly in the methods below
```suggestion
        SAFE.badger.set_guestlist_user_cap(guestlist, cap)
        SAFE.post_safe_tx()
```

Reply:
L26 will then need to be removed, since `post_safe_tx` doesnt necessarily end the script.

Reply:
ah actually same goes for all these funcs

Reply:
Good call, applying the change